### ISSUE TITLE
verbatimModuleSyntax  require + clsx not found

### ISSUE THREAD

**Issue by Sheepux** [orefalo/svelte-virtuallists#21]
>         "vite": "^5.4.10",
>         "svelte": "^5.1.3",
>        "svelte-check": "^4.0.5",
>        "svelte-preprocess": "^6.0.3",
>        "svelte-virtuallists": "^1.3.9",

I just installed the lib via npm (not a fresh project)
And suddently my svelte build is encountering issues

```
 The TypeScript option verbatimModuleSyntax is now required when using Svelte files with lang="ts". Please add it to your tsconfig.json.

X [ERROR] Could not resolve "clsx"
    ../node_modules/svelte-virtuallists/dist/VirtualListNew.svelte:9:17:
      9 │ import clsx from 'clsx';

  You can mark the path "clsx" as external to exclude it from the bundle, which will remove this
  error and leave the unresolved path in the bundle.
```

1. To solve clsx i had to declare it as devdependencies in my project
I still don't know what's the actual issue

2. The verbatim issue is an issue on svelte preprocess part
Related to https://github.com/sveltejs/svelte-preprocess/issues/643 
Caused by https://github.com/sveltejs/language-tools/issues/2391
(this could help someone having the issue)
I removed svelte preprocess  (as I was almost not using it) and the issue disapeared

**Comment by orefalo** [comment]
tks for reporting this issue

interesting...
clsx is the lib I use to build class and styles. I wonder why npm won't bundle it.

**Comment by orefalo** [comment]
would you have a test project to reproduce the issue.

I am using the library as part of another project https://github.com/orefalo/svelte-treeviews
  and it works perfectly

**Comment by ronny1982** [comment]
It works in your other project because `clsx` was added as dev dependency.
Consider adding `clsx` as dependency or at least peer dependency in `svelte-virtuallists`.

**Comment by orefalo** [comment]
will fix

**Comment by Sheepux** [comment]
fixed by v1.3.10 (closing the issue)

Thanks for the quick response :)

As for the svelte-preprocess issue there were some other solution that i didn't go through:
https://github.com/sveltejs/svelte-preprocess/issues/643#issuecomment-2217721286
